src/test/java/spoon/test/query_function/VariableReferencesTest.java

```diff
@@ -111,8 +116,8 @@ void checkKey(int key, CtElement ele) {
 			return false;
 		}).list();
 		assertFalse(context.unique.isEmpty());
-		assertEquals("Only these keys were found: "+context.unique.keySet(), context.maxKey, context.unique.size());
-		assertEquals("AllLocalVars#maxValue must be equal to maximum value number ", (int)getLiteralValue((CtVariable)modelClass.filterChildren(new NamedElementFilter<>(CtVariable.class,"maxValue")).first()), context.maxKey);
+		assertEquals(context.maxKey, context.unique.size(), "Only these keys were found: " + context.unique.keySet());
+		assertEquals(((int) (getLiteralValue(((CtVariable) (modelClass.filterChildren(new NamedElementFilter<>(CtVariable.class, "maxValue")).first()))))), context.maxKey, "AllLocalVars#maxValue must be equal to maximum value number ");

```

Comment:
Ugh, sniper printing of casts at work here (see https://github.com/INRIA/spoon/issues/3809#issuecomment-795342425). Could you manually remove the redundant parentheses?